src/Group.php

```diff
@@ -119,48 +119,62 @@ public function cleanDBonPurge()
         Rule::cleanForItemCriteria($this, '_groups_id%');
     }
 
-
     public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     {
+        /**  @var \DBmysql $DB
+         * @var array $CFG_GLPI
+         */
+        global $DB, $CFG_GLPI;
 
-        if (!$withtemplate && self::canView()) {
-            $nb = 0;
-            switch ($item->getType()) {
-                case 'Group':
-                    /** @var Group $item */
-                    $ong = [];
-                    if ($_SESSION['glpishow_count_on_tabs']) {
-                        $nb = countElementsInTable(
-                            $this->getTable(),
-                            ['groups_id' => $item->getID()]
-                        );
-                    }
-                    $ong[4] = self::createTabEntry(__('Child groups'), $nb);
+        if ($withtemplate || !self::canView()) {
+            return '';
+        }
 
-                    if ($item->getField('is_itemgroup')) {
-                        $ong[1] = __('Used items');
-                    }
-                    if ($item->getField('is_assign')) {
-                        $ong[2] = __('Managed items');
+        $ong = [];
+        if ($item instanceof Group) {
```

Comment:
Why did you switch from ```switch``` case to ```if```?

Reply:
Because with a switch case, i've a lot of error / warning like :
![image](https://github.com/user-attachments/assets/bf74936d-173f-46da-a8f0-d0457f77a1d5)

Reply:
PHPStan issues must not be resolved using a more complicated code.

A better way to do is something like:
```
            switch (get_class($item)) {
                case Group::class:
```